components/Alerts/alerts.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>replit</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link href="alerts.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+    <div class="Alerts-container">
+        <h1>Alerts</h1>
+        <h3>Alerts are used to attract user's attention for important information without interrupting the user's flow.
+        </h3>
```

Comment:
use semantics html like section, nav etc

Reply:
okkkk i thanks for suggestion i will try to include semantics in projects